src/models/enums/QuestionType.py

```diff
@@ -0,0 +1,7 @@
+import enum
+
+
+class QuestionType(enum.Enum):
+    ignore = 'ignore'
+    essay = 'essay'
+    multiple_choice = 'multipleChoice'
```

Comment:
that's ugly, can you auto it pls and just use the name

Reply:
Done